Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -1905,7 +1918,8 @@ internal unsafe void OnUpdate(InputUpdateType updateType, int eventCount, IntPtr
             var buffersToUseForUpdate = updateType;
 #if UNITY_EDITOR
             gameIsPlayingAndHasFocus = InputConfiguration.LockInputToGame ||
-                (UnityEditor.EditorApplication.isPlaying && Application.isFocused);
+                (UnityEditor.EditorApplication.isPlaying && Application.isFocused) ||
+                ((updateMask & InputUpdateType.IgnoreFocus) != 0);
```

Comment:
The added check seems like it should be outside the `#if UNITY_EDITOR` block.

Reply:
It's good inside the UNITY_EDITOR block, because outside the editor we rely on the native updates being called.  It's by default running.